packages/core/http/core-http-server/src/versioning/example.ts

```diff
@@ -7,22 +7,20 @@
  */
 
 import { schema } from '@kbn/config-schema';
-import type { IRouter, RequestHandlerContextBase } from '@kbn/core-http-server';
-import type { VersionHTTPToolkit } from './version_http_toolkit';
+import type { RequestHandlerContextBase } from '../..';
+import { VersionedRouter } from './types';
 
 interface MyCustomContext extends RequestHandlerContextBase {
   fooService: { create: (value: string, id: undefined | string, name?: string) => Promise<void> };
 }
-const vtk = {} as unknown as VersionHTTPToolkit;
-const router = {} as unknown as IRouter<MyCustomContext>;
-
-const versionedRouter = vtk.createVersionedRouter({ router });
+const versionedRouter = {} as unknown as VersionedRouter<MyCustomContext>;
```

Comment:
Now that we have an implementation, should we get rid of this file, or convert it to documentation or TSDoc?